pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -529,10 +534,15 @@ def select_accelerator(self) -> Accelerator:
             acc_cls = CPUAccelerator
         # as precision_plugin is dependent on training_type_plugin, make sure
         # that we first select training_type_plugin, then precision_plugin
-        return acc_cls(
+        accelerator = acc_cls(
             training_type_plugin=self.training_type_plugin,
             precision_plugin=self.precision_plugin,
         )
+        # transfer ownership of the plugins to the accelerator
+        self._training_type_plugin = proxy(self.training_type_plugin)
+        self._precision_plugin = proxy(self.precision_plugin)
```

Comment:
No we can't since the first have to be given to the accelerator (so that we have a hard reference somewhere). Otherwise we end up with no reference at all which may result in these objects being deleted too early.